recipes/base64/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
+
+
+class CppcodecConan(ConanFile):
+    name = "base64"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aklomp/base64"
+    description = "Fast Base64 stream encoder/decoder"
+    topics = ("base64", "codec")
+    exports_sources = ["patches/**"]
```

Comment:
```suggestion
    exports_sources = "patches/**", "CMakeLists.txt"
```
Please add a base cmake wrapper script with the following:
```cmake
cmake_minimum_required(VERSION 2.8.12)
project(cmake_wrapper)

include(conanbuildinfo.cmake)
conan_basic_setup()

add_subdirectory(source_subfolder)
```